ocaml/idl/datamodel_host.ml

```diff
@@ -1459,6 +1459,18 @@ let host_query_ha = call ~flags:[`Session]
     ~allowed_roles:_R_POOL_ADMIN
     ()
 
+  let copy_primary_host_certs = call
+    ~name:"copy_primary_host_certs"
+    ~in_oss_since:None
+    ~in_product_since:rel_next
+    ~doc:"useful for secondary slaves that are missing some certs"
+    ~params:[
+      Ref _host, "host", "the primary host will copy its set of trusted certs onto this host";
```

Comment:
The host updates its trusted certificates by receiving them from the primary host in the pool.